src/dependency-manager/package.json

```diff
@@ -17,13 +17,13 @@
   "homepage": "https://github.com/architect-team/architect-cli",
   "license": "GPL-3.0",
   "dependencies": {
-    "@types/shell-quote": "^1.6.2",
     "axios": "^0.19.0",
     "class-transformer": "^0.2.3",
     "class-validator": "^0.12.2",
     "dotenv-expand": "^5.1.0",
     "fs-extra": "^8.1.0",
     "js-yaml": "^3.13.1",
+    "mustache": "^4.0.1",
```

Comment:
Need to update the package-lock.json. Make sure you remove the node modules dir for dependency-manager after installing

Reply:
Done, good call